tasks/openssl.yml

```diff
@@ -0,0 +1,114 @@
+---
+
+- name: create certificate directory structure
+  file:
+    path: "{{ cert_path }}"
+    state: directory
+    owner: root
+    group: root
+    mode: 0775
+
+- name: create ca private key
+  command: "
+    openssl genrsa
+    -out {{ cert_path }}/ca_key.pem 4096"
```

Comment:
Write that on one line.
This command is not idempotent! E.g. use `creates` parameter from module [command](https://docs.ansible.com/ansible/latest/modules/command_module.html#parameters)